scss/_grid.scss

```diff
@@ -12,6 +12,46 @@
   }
 }
 
+$enable-cssgrid: true !default;
+
+@mixin make-cssgrid($columns: $grid-columns, $breakpoints: $grid-breakpoints) {
+  @each $breakpoint in map-keys($breakpoints) {
+    $infix: breakpoint-infix($breakpoint, $breakpoints);
+
+    @include media-breakpoint-up($breakpoint, $breakpoints) {
+      @if $columns > 0 {
+        @for $i from 1 through $columns {
+          .g-col#{$infix}-#{$i} {
+            grid-column: auto / span $i;
+          }
+        }
+
+        // `$columns - 1` because offsetting by the width of an entire row isn't possible
+        @for $i from 0 through ($columns - 1) {
+          .g-start#{$infix}-#{$i} {
+            grid-column-start: $i;
+          }
+        }
+      }
+    }
+  }
+}
+
+@if $enable-cssgrid {
+  .grid {
+    --rows: 1;
+    --columns: #{$grid-columns};
+    --gap: #{$grid-gutter-width};
```

Comment:
We could set those as fallback values, instead of explicitly declaring them.

Reply:
Indeed. Also, these should be prefixed.